lib/jwtRedisSession.js

```diff
@@ -1,22 +1,28 @@
-const _ = require("lodash"),
-	jwt = require("jsonwebtoken"),
-	utils = require("./utils");
+const { reduce, extend } = require("lodash");
```

Comment:
If you can, use specific _lodash_ imports like following:

https://www.npmjs.com/package/lodash.reduce

Reply:
https://www.npmjs.com/package/lodash.reduce this is a separated package. I couldn't find a package for "extend" lodash utility, so I've kept the "lodash" dependency and I've imported each utility in a separated sentence.